src/dbos-executor.ts

```diff
@@ -647,6 +667,29 @@ export class DBOSExecutor implements DBOSExecutorContext {
     }
     const wConfig = wInfo.config;
 
+    // Compatibility with the old way of calling workflows, which would include a parentCtx
+    const pctx = getCurrentContextStore();
+    const passContext = wInfo.registration?.passContext ?? true;
+    if (!passContext && pctx) {
+      const span = this.tracer.startSpan(
+        wf.name,
+        {
+          operationUUID: workflowUUID,
+          operationType: OperationType.WORKFLOW,
```

Comment:
This operationType, should be defined by the workflow "parent". So for instance, HTTP handler.